src/app/src/test/kotlin/org/imperial/mrc/hint/integration/HintApplicationTests.kt

```diff
@@ -37,8 +37,7 @@ class HintApplicationTests : SecureIntegrationTests()
     {
         val response = testRestTemplate.getForEntity<String>("/metrics")
         assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
-        assertThat(response.body).isEqualTo("running 1\nbusy_workers 0\nidle_workers 2\n" +
-                "paused_workers 0\nexited_workers 0\nlost_workers 0\nlive_workers 2")
+        assertThat(response.body).contains("running 1")
```

Comment:
This was not always the case, sometimes the workers were busy when this endpoint was hit. The more specific message is tested in the unit tests, so here just check we're accessing the endpoint.